### DISCUSSION TITLE
ServiceStack v6 - Route starting with /api produces NotImplementedException

### DISCUSSION THREAD

**Discussion by Codeh4ck** [ServiceStack/Discuss#14] [Q&A] [answered]
### Greetings.

I started a new project with **ServiceStack v6**. I noticed that when I use routes starting with `/api`, they will produce a `NotImplementedException` when called. I believe this is due to the new API Explorer UI implemented on ServiceStack v6. 

Is there any way to disable this? I want my `/api` routes to work as they used to work on ServiceStack **5.13.2**
Or perhaps change the /api prefix to something else would also be fine by me, API Explorer UI seems like a feature I would like to use. 

### Notes:
Removing `/api` from my route will make it work just fine.

**Working example:**

```c#
appHost.Routes.Add<JoinChatRequest>("/chat/join", ApplyTo.Get);
```

**Example producing a `NotImplementedException` when called:**

```c#
appHost.Routes.Add<JoinChatRequest>("/api/chat/join", ApplyTo.Get);
```

**Comment by mythz** [comment, marked as answer]
This is due to the [JSON /api pre-defined route](https://docs.servicestack.net/routing#json-api-pre-defined-route) that was [added in v6.0](https://docs.servicestack.net/releases/v6#new-api-route) which can be explicitly [can be disabled with](https://docs.servicestack.net/routing#disable-api-route):

```csharp
ConfigurePlugin<PredefinedRoutesFeature>(feature => feature.JsonApiRoute = null);
```

**Reply by Codeh4ck** [reply]
Hats off to you @mythz. Exceptional support as always, thank you. I missed that bit in the documentation.
